StandardRecord/SRSSDHits.cxx

```diff
@@ -17,7 +17,9 @@ namespace caf
     Time(std::numeric_limits<float>::signaling_NaN()),
     ADC(std::numeric_limits<float>::signaling_NaN()),
     TrigNum(std::numeric_limits<float>::signaling_NaN()),
-    Row(std::numeric_limits<float>::signaling_NaN())
```

Comment:
Did you intend to remove Row?

Reply:
I absolutely did not intend to remove row, but I added it back in now